tensorflow/python/keras/utils/vis_utils.py

```diff
@@ -141,6 +192,10 @@ def model_to_dot(model,
 
   # Create graph nodes.
   for i, layer in enumerate(layers):
+    if (layer_range is not None) and\
```

Comment:
Same here, please don't use \ to break the long line.

Reply:
done